agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/service/GeneDiseaseAnnotationService.java

```diff
@@ -117,7 +133,9 @@ public Map<Gene, List<DiseaseAnnotation>> getOrthologousGeneDiseaseAnnotations(M
 					continue;
 				}
 				// create orthologous DAs for each focus DA
-				focusDiseaseAnnotations.forEach(focusDiseaseAnnotation -> {
+				
+				for(DiseaseAnnotation focusDiseaseAnnotation: focusDiseaseAnnotations) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'for' is not followed by whitespace.
<!-- __reviewdog__:ChAxMjI1ZjM1NzMwNzIwNTAwEgpjaGVja3N0eWxl -->